console/src/components/Editor/FullTablesEditor.tsx

```diff
@@ -10,39 +10,55 @@ import { withProject } from './Editor'
 import { withDefaultWidgets } from '@starwhale/core/widget'
 import StoreUpdater from '@starwhale/core/store/StoreUpdater'
 
+function groupBy(names: string[]) {
+    const m = {}
+    names?.forEach((name) => {
+        const key = name.split('/')?.[5]
+        // group by arr the fifth element
+        m[key] = m[key] || []
+        m[key].push(name)
+    })
+    return m
+}
```

Comment:
Is it better to use lodash's groupBy?
(this function can not handle keys like `toString`, `valueOf`...)

Reply:
check